app/plugins/router.js

```diff
@@ -1,31 +1,44 @@
-const config = require('../config')
+import { config } from '../config/index.js'
+import { accessibilityRouteHandlers } from '../routes/accessibility.js'
+import { assetsRouteHandlers } from '../routes/assets.js'
+import { cookieHandlers } from '../routes/cookies.js'
+import { healthHandlers } from '../routes/health.js'
+import { oldWorldStartRouteHandlers } from '../routes/index.js'
+import { privacyPolicyRouteHandlers } from '../routes/privacy-policy.js'
+import { signinRouteHandlers } from '../routes/signin-oidc.js'
+import { oldWorldEntryRouteHandlers } from '../routes/old-world-entry.js'
+import { indexRouteHandlers } from '../routes/endemics/index.js'
+import { numbersRouteHandlers } from '../routes/endemics/numbers.js'
+import { claimMultipleRouteHandlers } from '../routes/endemics/you-can-claim-multiple.js'
+import { reviewsRouteHandlers } from '../routes/endemics/reviews.js'
+import { declarationRouteHandlers } from '../routes/endemics/declaration.js'
+import { checkDetailsRouteHandlers } from '../routes/endemics/check-details.js'
+import { timingsRouteHandlers } from '../routes/endemics/timings.js'
 
-let routes = [].concat(
-  require('../routes/accessibility'),
-  require('../routes/assets'),
-  require('../routes/cookies'),
-  require('../routes/guidance-for-farmers'),
-  require('../routes/healthy'),
-  require('../routes/healthz'),
-  require('../routes/index'),
-  require('../routes/privacy-policy'),
-  require('../routes/vet-technical'),
-  require('../routes/signin-oidc')
-)
+let routes = [
+  ...accessibilityRouteHandlers,
+  ...assetsRouteHandlers,
+  ...cookieHandlers,
+  ...healthHandlers,
+  ...oldWorldEntryRouteHandlers,
+  ...oldWorldStartRouteHandlers,
+  ...privacyPolicyRouteHandlers,
+  ...signinRouteHandlers
+]
 
 if (config.endemics.enabled) {
```

Comment:
Potentially worth removing the `let routes = ...` stuff, as endemics is always enabled? Could delete 10ish lines of code

Reply:
I was planning to leave the removal of endemics flag to a separate later PR, just because it would balloon this one even more, so based on that thought there was no point removing any individual little bits depending on the flag